simpa_examples/create_a_custom_digital_device_twin.py

```diff
@@ -41,12 +41,12 @@ def __init__(self):
 
     positions = device.get_detection_geometry().get_detector_element_positions_accounting_for_device_position_mm()
     detector_elements = device.get_detection_geometry().get_detector_element_orientations()
-    positions = np.round(positions/settings[Tags.SPACING_MM]).astype(int)
+    positions = sp.round_away_from_zero(positions/settings[Tags.SPACING_MM])
     xdim, zdim, ydim, xdim_start, xdim_end, ydim_start, ydim_end, zdim_start, zdim_end = compute_image_dimensions(device.get_detection_geometry().field_of_view_extent_mm, settings[Tags.SPACING_MM], Logger())
 
     import matplotlib.pyplot as plt
     from matplotlib.patches import Rectangle
-    plt.gca().add_patch(Rectangle((xdim_start ,ydim_start), xdim, -ydim, linewidth=1, edgecolor='r', facecolor='r', alpha=.5))
+    plt.gca().add_patch(Rectangle((xdim_start + 50/2, ydim_start), xdim + 50/2, -ydim, linewidth=1, edgecolor='r', facecolor='r', alpha=.5))
```

Comment:
What is the purpose of the `+50/2` ? Now the rectangle of the FOV is 25 pixels larger to the left than the sensor array

Reply:
you're right, I corrected it